std/process.d

```diff
@@ -1709,6 +1834,25 @@ void kill(Pid pid, int codeOrSignal)
     assertThrown!ProcessException(kill(pid));
 }
 
+@system unittest // wait() and kill() detached process
+{
+    import core.thread;
+    import std.exception : assertThrown;
+    TestScript prog = "exit 0";
+    auto pid = spawnProcess([prog.path], null, Config.detached);
+    /*
+    This sleep is needed because we can't wait() for detached process to end
+    and therefore TestScript destructor may run at the same time as /bin/sh tries to start the script.
+    This leads to the annoying message like "/bin/sh: 0: Can't open /tmp/std.process temporary file" to appear when running tests.
+    It does not happen in unittests with non-detached processes because we always wait() for them to finish.
+    */
+    Thread.sleep(dur!"msecs"(5));
```

Comment:
I used 5 msecs, because there's the same sleep in the other unittest under version(Android), having a comment about Android killing sleeping applications.